mypy/main.py

```diff
@@ -175,8 +173,6 @@ def parse_version(v: str) -> Tuple[int, int]:
     parser.add_argument('--strict-optional', action='store_true',
                         dest='special-opts:strict_optional',
                         help="enable experimental strict Optional checks")
-    parser.add_argument('-f', '--dirty-stubs', action='store_true',
-                        help="don't warn if typeshed is out of sync")
```

Comment:
Please keep the flag around. It's okay to print a warning message when it's used, but it should not fail. Too many people may have automation that runs `mypy -f`. See how we're treating `--silent` and `--use-python-path` below.